src/inspector_agent.cc

```diff
@@ -338,11 +345,58 @@ AgentImpl::~AgentImpl() {
   data_written_ = nullptr;
 }
 
+void InspectorCallSequently(const v8::FunctionCallbackInfo<v8::Value>& args) {
+  static bool in_sequently_call = false;
+  bool call_inspector = !in_sequently_call;
+  in_sequently_call = true;
+
+  v8::Local<v8::Context> context = args.GetIsolate()->GetCurrentContext();
+
+  v8::Local<v8::Array> functions = args.Data().As<v8::Array>();
+  v8::Local<v8::Function> inspectorMethod =
+    functions->Get(context, 0).ToLocalChecked().As<v8::Function>();
+  v8::Local<v8::Function> nodeMethod =
+    functions->Get(context, 1).ToLocalChecked().As<v8::Function>();
+
+  v8::Local<v8::Value>* info = new v8::Local<v8::Value>[args.Length()];
+  for (int i = 0; i < args.Length(); ++i) {
+      info[i] = args[i];
+  }
+
+  if (call_inspector) {
+    inspectorMethod->Call(context, args.Holder(), args.Length(), info);
+  }
+
+  nodeMethod->Call(context, args.Holder(), args.Length(), info);
+
+  in_sequently_call = false;
+}
+
+void InspectorSequentally(const v8::FunctionCallbackInfo<v8::Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (args.Length() != 2) {
+    return env->ThrowError("Bad argument.");
+  }
+
+  v8::Local<v8::Array> array = v8::Array::New(env->isolate());
+  for (int i = 0; i < args.Length(); ++i) {
+    if (!array->Set(env->context(), i, args[i]).FromMaybe(false)) {
+      continue;
```

Comment:
I think there is nothing to do but abort if setting the property fails.  I'd write this as:
```c++
CHECK(array->Set(env->context(), i, args[i]).FromJust());
```